File: ICPC/Calico 2025/Competition/gta6/templates/gta6_template.cpp
#include <iostream>

using namespace std;
int Y_0 = 2026;
int M_0 = 11;
int D_0 = 19;

/**
 * E: Name of event
 * Y: Year of event
 * M: Month of event
 * D: Day of event
 */
 string solve(string E, int Y, int M, int D) {
    // YOUR CODE HERE
    string before = "we got " + E + " before gta6";
    string after = "we got gta6 before " + E;

    return (Y < Y_0 || (Y == Y_0 && M < M_0) || (Y == Y_0 && M == M_0 && D < D_0) ? before : after;
}

int main() {
    int T;
    cin >> T;
    cin.ignore();
    for (int i = 0; i < T; i++) {
        string E;
        getline(cin, E);
        int D, M, Y;
        cin >> Y >> M >> D;
        cin.ignore();
        cout << solve(E, Y, M, D) << '\n';
    }
}

File: ICPC/Calico 2025/Competition/gta6/templates/gta6_template.py
Y_0 = 2026
M_0 = 11
D_0 = 19

def solve(E: str, D: int, M: int, Y: int) -> str:

    """

    E: The name of the event
    Y: Year
    M: Month
    D: Day
    """
    # YOUR CODE HERE
    before = 'we got ' + E + ' before gta6'
    after = 'we got gta6 before ' + E

    return before if Y < Y_0 or (Y == Y_0 and M < M_0) or (Y == Y_0 and M == M_0 and D < D_0) else after

def main():
    T = int(input())
    for _ in range(T):
        E = input()
        temp = input().split()
        Y, M, D = int(temp[0]), int(temp[1]), int(temp[2])
        print(solve(E, Y, M, D))

if __name__ == 'main':
    main()